content/en/events/2023/kcsna/_index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Contributor Summit North America 2023"
+type: docs
+weight: 1
+description: >
+  Kubernetes Contributor Summit North America, hosted in Chicago, Illinois.
+
+---
+
+
+### About
+
+The North America Summit will be a lightweight event taking place on November 6th
+in Chicago, Illinois alongside 
+<a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" rel="noopener noreferrer" target="_blank">KubeCon</a>
+at the
+<a href="https://www.hyatt.com/en-US/hotel/illinois/hyatt-regency-mccormick-place/chimc" rel="noopener noreferrer" target="_blank">Hyatt Regency McCormick Place</a>.
+
+The Summit will be a loosely organized event with a [relaxed schedule](/events/2023/kcsna/schedule),
+ including a Steering AMA, several hours of Unconference Sessions, 
+potential SIG and WG meetings, and a Documentation Sprint.  This will be followed
+by a [Contributor Social](/events/2023/kcsna/social) with games and refreshments.
+
+This page will be updated as more activities are scheduled. If you have
+questions or comments, please reach out to the Summit Staff by emailing
+community@kubernetes.io or asking in the
+<a href="https://kubernetes.slack.com/messages/contributor-summit" rel="noopener noreferrer" target="_blank">#contributor-summit</a>
+slack channel.
+
+[location]: /events/2023/kcsna/location/
+
+### Recordings
+
+The links to recordings and photos will be published after the event.
+
+
+### Call for Papers
+
+TODO: update the date and CFP link below:
+A [CFP] for sessions will be open through Friday, October 6th.
+
+[CFP]: https://docs.google.com/forms/d/1z7TnjixN9fiuQulF-EU9rw7_U9I-I8HNOv28IAFsHZc/viewform
```

Comment:
All the TODOs can probably just be removed until we have more info, or replaced with 'TBD'

Reply:
Thank you! I'll remove old links/info and put TBD until we have more information, would that be OK?

Reply:
We now have proper links for Reg and CFP